src/transformers/models/auto/modeling_auto.py

```diff
@@ -439,6 +439,14 @@
     ]
 )
 
+MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING_NAMES = OrderedDict(
+    [
+        # Model for Audio Classification mapping
+        ("wav2vec2", "Wav2Vec2ForSequenceClassification"),
```

Comment:
Should we quickly rename them to `...ForAudioClassification`? To avoid confusion when using pipelines and be more in line with `...ForImageClassification`

Reply:
I think it's fine. IMO `ForSequenceClassification` is totally fine...(maybe someone wants to use the model for stock market prediction, classification, ....). We'll do a release tomorrow so not sure we can change it after anymore